toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -155,20 +155,18 @@
           "value": "{color.gray.05}"
         },
         "deemphasized": {
-          "light": {
-            "value": "color-mix(in srgb, currentColor 69%, transparent)"
-          },
-          "dark": {
-            "value": "color-mix(in srgb, currentColor 75%, transparent)"
-          }
+          "value": "light-dark(color-mix(in srgb, currentColor 69%, transparent),color-mix(in srgb, currentColor 75%, transparent))"
```

Comment:
I'm guessing this and the other `light-dark` reference are leftover from some experimenting? Again let's just modify this JSON so that it's in the format we need it to be in for the transform to work